services/datamanager/builtin/sync_test.go

```diff
@@ -403,7 +410,9 @@ func TestArbitraryFileUpload(t *testing.T) {
 			// Write file to the path.
 			var fileContents []byte
 			for i := 0; i < 1000; i++ {
-				fileContents = append(fileContents, []byte("happy cows come from california\n")...)
+				if tc.name != emptyFileTestName {
+					fileContents = append(fileContents, []byte("happy cows come from california\n")...)
+				}
```

Comment:
might be nice to pull out this for loop into a helper called `populateFileContents` or something and only call it `if tc.name != emptyFileTestName` so you don't loop 1000 times for nothing

Reply:
yeah I agree - just fixed it!